README.md

```diff
@@ -52,4 +52,37 @@ Prerequisites:
     * Enter password
 * `python manage.py runserver`
 * Go to http://localhost:8000/staff
-* Sign in as the superuser you created above
\ No newline at end of file
+* Sign in as the superuser you created above
+
+### Configuring Factom Blockchain Connection
+
+In order to submit evidences to the Factom Blockchain, you will need access to:
+* running `factomd` and `factom-walletd` instances
+* an entry credit address to fund Factom Entries
+* a 32 byte private seed for producing digital signatures using the Ed25519 algorithm
```

Comment:
Would be nice to just add a one-liner that generates one.

Reply:
That generates a private seed or an entry credit address?

Reply:
Your code does the latter, right? So the one-liner would do the former.